#60: Fixed release build command

tomuben opened the pull request:
fixes #60

tomuben committed:
#60: Fixed release build command

tkilias commented:
Maybe make default_config private

https://github.com/exasol/script-languages-developer-sandbox/blob/6c3912596a71e1733fc737d66f7c57e9e7290d9a/exasol_script_languages_developer_sandbox/lib/config.py#L5

tomuben committed:
1. Made default_config protected in config.py
2. Fixed also install_dependencies.py
3. changed test_config() in conftest.py such that it uses the config object, not the dictionary
